tests/profile/ProfileUiTest.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class ProfileUiTest extends PHPUnit_Framework_TestCase {
+    public function testProfileContainsUis() {
+        $basePath = dirname(dirname(__DIR__));
+        $doc = new DOMDocument();
+        $doc->load("$basePath/profile/rwahs.xml");
+        $xpath = new DOMXPath($doc);
+        $this->assertEquals(14, $xpath->query('//userInterfaces/userInterface')->length, 'The number of user interfaces should match');
```

Comment:
I reckon if we're going to bother with this test, we should check that all the expected UIs are present by checking the `code` for each one.  You could probably do this with XPath:

    $this->assertEquals(1, $xpath->query('//userInterfaces/userInterface[code="the_code"]'), 'The `the_code` user interface should exist');